test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -2708,29 +2824,35 @@ odp_testinfo_t crypto_suite[] = {
 
 /* Suite names */
 #define ODP_CRYPTO_SYNC_INP         "odp_crypto_sync_inp"
-#define ODP_CRYPTO_ASYNC_INP        "odp_crypto_async_inp"
+#define ODP_CRYPTO_ASYNC_PLAIN_INP  "odp_crypto_async_plain_inp"
+#define ODP_CRYPTO_ASYNC_SCHED_INP  "odp_crypto_async_sched_inp"
 #define ODP_CRYPTO_PACKET_SYNC_INP  "odp_crypto_packet_sync_inp"
-#define ODP_CRYPTO_PACKET_ASYNC_INP "odp_crypto_packet_async_inp"
+#define ODP_CRYPTO_PACKET_ASYNC_PLAIN_INP "odp_crypto_packet_async_plain_inp"
+#define ODP_CRYPTO_PACKET_ASYNC_SCHED_INP "odp_crypto_packet_async_sched_inp"
```

Comment:
All these defines can be removed, as only place those are used is just under here.

Reply:
Ack.  Removed the defines in v5.